Remotion/ObjectBinding/Web/UI/Controls/BocValidationErrorIndicatorColumnDefinition.cs

```diff
@@ -0,0 +1,43 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Web.UI.WebControls;
+using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation.Rendering;
+using Remotion.ServiceLocation;
+
+namespace Remotion.ObjectBinding.Web.UI.Controls
+{
+  /// <summary>
+  /// A column definition for displaying a validation error indicator for a row.
+  /// </summary>
+  public class BocValidationErrorIndicatorColumnDefinition : BocColumnDefinition
```

Comment:
this is part of a separate PR? I think you may have the wrong base right now?

Reply:
The proper implementation for this class will be/ is added in a separate issue, we need the knowledge of the type here already though. At least that's what I think this was about. @a-ctor?

Reply:
@Frazerus okay, thanks.

Reply:
My last answer was not 100% correct. This does belong to this changeset, it does however require the BoValidationErrorIndicatorColumnRenderer from RM-8802. Will change this to a nullcolumnrenderer for the time being.